aws/resource_aws_glue_crawler.go

```diff
@@ -529,7 +594,8 @@ func resourceAwsGlueCrawlerUpdate(d *schema.ResourceData, meta interface{}) erro
 
 	if d.HasChanges(
 		"catalog_target", "classifiers", "configuration", "description", "dynamodb_target", "jdbc_target", "role",
-		"s3_target", "schedule", "schema_change_policy", "security_configuration", "table_prefix", "mongodb_target") {
+		"s3_target", "schedule", "schema_change_policy", "security_configuration", "table_prefix", "mongodb_target",
+		"lineage_configuration", "recrawl_policy") {
```

Comment:
Suggest changing to `d.HasChangesExcept("tags")` so that we don't miss any non-`tags` attributes in future.

Reply:
Nice, wasnt aware of this one

Reply:
See https://github.com/hashicorp/terraform-provider-aws/issues/16627.

Reply:
Added change and retested. Thanks for this.